src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/exceptions/ApplicationExceptionHandler.java

```diff
@@ -45,13 +52,18 @@
     @Autowired
     private ExceptionTranslator exceptionTranslator;
 
+    @Autowired
+    @Lazy
+    private AuditEventLoggingFacade auditEventLoggingFacade;
```

Comment:
Good point! I don't think it works. It was a remnant of older implementation where AuditEventLoggingFacade was a request scoped bean, and injecting it would break when used from outside of request scope (like when exceptions happen during bootup)

AuditEventLoggingFacade is now a regular bean so I removed those Lazy annotations.  Now only RequestScopedAuditEventData and RequestScopedLoggedAuditEvents are request scoped beans, and those are injected as lazy to RequestScopedAuditDataHolder